how-to/customize-workspace/client/src/framework/workspace/notifications.ts

```diff
@@ -56,11 +58,18 @@ export async function deregister() {
 		const settings = await getSettings();
 		const notificationPlatform = settings.notificationProvider;
 		if (notificationPlatform !== undefined) {
-			await deregisterPlatform(notificationPlatform.id);
-			logger.info("Unregistered platform notifications");
+			logger.info("Deregister platform from notifications");
+			deregisterPlatform(notificationPlatform.id)
```

Comment:
This probably shouldnt be background process, it is safer for the close down process to wait for it to happen, otherwise it could get processed when the platform is already torn down

Reply:
Will discuss.